<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChoosePlayers</class>
 <widget class="QDialog" name="ChoosePlayers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="Check2Players">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2 players</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="Check3Players">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>3 players</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Check4Players">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>4 players</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StartGameButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
